packages/auth/src/Login/LoginWithPassword.tsx

```diff
@@ -10,10 +10,12 @@ import {
   FButton,
   FInput,
   FFormik,
+  Icon,
 } from '@frontegg/react-core';
 import { useAuth } from '../hooks';
-
 const { Formik } = FFormik;
+import Visibility from '@material-ui/icons/Visibility';
```

Comment:
- Use Icon from @frontegg/react-core
- add Visibility/VisibilityOff to the IconNames map
- move passwordType logic to the input item inside @frontegg/react-core. if `props.type === 'password'` will add this logic to the components.